browser-app/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Can we have licence headers in package.json? Can you check with the other che-specific Theia plugins

Reply:
I've checked and we aren't able to have license headers here/other extensions don't have it. JSON doesn't support comments.